util/util.go

```diff
@@ -112,7 +112,7 @@ func ResolveName(name string, firstRegistry string, sc *types.SystemContext, sto
 	}
 	for _, registry := range searchRegistries {
 		if !registry.Blocked {
-			registries = append(registries, registry.URL)
+			registries = append(registries, registry.Location)
 		}
 	}
 	searchRegistriesAreEmpty := len(registries) == 0
```

Comment:
DId you mean for these four files to be changed?  I'd really rather just have the vendor bump for image/storage to be in this PR, then the new bolt stuff in a separate one.

Reply:
These changes were required by the new version of containers/image.

Reply:
Okey Doke, thanks for the ptr.